addons/sale_product_configurator/static/src/js/product_configurator_modal.js

```diff
@@ -79,11 +79,8 @@ var OptionalProductsModal = Dialog.extend(ServicesMixin, VariantMixin, {
             variant_values: self.rootProduct.variant_values,
             pricelist_id: self.pricelistId || false,
             add_qty: self.rootProduct.quantity,
-            kwargs: {
-                context: _.extend({
-                    'quantity': self.rootProduct.quantity
-                }, this.context),
-            }
+            quantity: self.rootProduct.quantity,
```

Comment:
Edit: I missed something, I thought it wasn't used, but it was (for `_get_contextual_price`, called in the qweb template rendered by `show_advanced_configurator`.
So you should keep the quantity, but integrate it in the rpc context directly (instead of giving it as rpc parameter).

Reply:
I don't understand where the `if 'kwargs' in kw and 'context' in kw['kwargs']:` is, could you tell me ?
What do you mean by integrating quantity in the rpc context ? Can I simply pass this in the rpc parameters ? `context: _.extend({'quantity': self.rootProduct.quantity}, this.context)`

Reply:
Oh, my bad, was looking at master (without noticing the part I was speaking was recent :man_facepalming:), forget that part of my remark and yes, passing the quantity in the context should work fine :+1: